.travis.yml

```diff
@@ -21,9 +21,9 @@ addons:
   postgresql: '9.4'
 before_install:
 - "echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc"
+- "[[ -f certs/v2_key.dev ]] || cp certs/v2_key.dev certs/v2_key"
```

Comment:
@brandondunne 
Unless I'm reading wrong, I think that suggestion will copy the v2_key only if its already exists.

I was going with copy the v2_key.dev to v2_key if it exists.